app/config/config.yml

```diff
@@ -76,6 +76,9 @@ doctrine:
             default:
                 connection: default
                 auto_mapping: false
+                metadata_cache_driver: array
+                query_cache_driver: array
+                result_cache_driver: array
```

Comment:
They don't add any logic, but are used like a placeholder, because that values will need to be updated ASAP for the final user. Much easier for them :)

What do you think?